Task/Tasks/TSKSelectorTask.h

```diff
@@ -77,7 +93,7 @@
      May not be NULL.
  @result A newly initialized TSKSelectorTask instance with the specified name, target, and action.
  */
-- (instancetype)initWithName:(NSString *)name target:(id)target selector:(SEL)selector;
+- (instancetype)initWithName:(nullable NSString *)name target:(id)target selector:(SEL)selector;
```

Comment:
Right, I think `target:(id)target` should remain nonnull, but up where the property is declared, I think it needs the `nullable` flag because it is weak. The object could be released